app/main.py

```diff
@@ -1,8 +1,34 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(
+            self,
+            name: str,
+            age: int
+    ) -> None:
+        self.name = name
+        self.age = age
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    for one_of_people in people:
+        Person.people[one_of_people["name"]] = \
+            Person(one_of_people["name"], one_of_people["age"])
+
+    for one_of_people in people:
+        if "wife" in one_of_people.keys():
+            for find_person in Person.people:
+                if one_of_people["wife"] == find_person:
+                    Person.people[one_of_people["name"]].wife = \
+                        Person.people[find_person]
+        if "husband" in one_of_people.keys():
+            for find_person in Person.people:
+                if one_of_people["husband"] == find_person:
+                    Person.people[one_of_people["name"]].husband = \
+                        Person.people[find_person]
```

Comment:
Try to do this without loop in loop. 
It would be much more readable solution. 

Consider, that you have people dict, where you can find any person by name. Try to get wife's name, check if it's not None, (the key might exist, but the value may be None)  and after that get the person object from the dict.